requirements/default.txt

```diff
@@ -1 +1,2 @@
 celery>=4.0,<5.0
+django-timezone-field==2.0
```

Comment:
This seems very restrictive to me (`>=` would be appropriate if you rely on code implemented in `django-timezone-field` 2.0).